### ISSUE TITLE
Documentation tasks for 0.5

### ISSUE THREAD

**Issue by robvdl** [lxdock/lxdock#122]
[ ] Document idmap changes
[ ] Document network timeout (extras)

**Comment by mtausig** [comment]
With #169 merged, we can check the second last box.

**Comment by robvdl** [comment]
Yeah we really should try to make the 0.5 release happen. I got so busy at work over the last year the lxdock project just didn't move (until recently again).

**Comment by robvdl** [comment]
See #184
